.github/workflows/deploy.yml

```diff
@@ -74,6 +89,21 @@ jobs:
         env:
```

Comment:
Above, in the `www` deploy to Netlify, can we change the `alias` to `ocw-hugo-themes-www-pr-${{ github.event.number }}`, to be more clear about what the deploy is?